src/widget/wfindonmenu.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <QAction>
+#include <QMenu>
+#include <QUrlQuery>
+
+class Track;
+
+class WFindOnMenu : public QMenu {
+    Q_OBJECT
+  public:
+    explicit WFindOnMenu(
+            QWidget* parent = nullptr);
+    ~WFindOnMenu() override = default;
+
+    enum class Services {
```

Comment:
Each value of this enum denotes a single service, not multiple. The type name is confusing.

Reply:
As long as this enum is not needed by consumers of the menu make it private or hide it in the .cpp file in an anonymous namespace.